### ISSUE TITLE
Show player's Team

### ISSUE THREAD

**Issue by dylanpiera** [dylanpiera/CodenameIndigo#49]
Allow players to view each other's team when the game is ready.

I'm not 100% sure if we decided to use this or not, do you recall @jdsaphir ?

**Comment by jdsaphir** [comment]
Once the tournament has started and teams can't be changed anymore, it should be possible.
Allows player to make sure the opponent isn't cheating. May want to ask DN for confirmation, just in case.

**Comment by dragonnataku** [comment]
yes, after the registration period is over they can see opponents' teams
